README.md

```diff
@@ -0,0 +1,564 @@
+[![jazzband](https://jazzband.co/static/img/badge.svg)](https://jazzband.co/)
+[![pypi](https://img.shields.io/pypi/v/pip-tools.svg)](https://pypi.org/project/pip-tools/)
+[![pyversions](https://img.shields.io/pypi/pyversions/pip-tools.svg)](https://pypi.org/project/pip-tools/)
+[![pre-commit](https://results.pre-commit.ci/badge/github/jazzband/pip-tools/main.svg)](https://results.pre-commit.ci/latest/github/jazzband/pip-tools/main)
+[![buildstatus-gha](https://github.com/jazzband/pip-tools/workflows/CI/badge.svg)](https://github.com/jazzband/pip-tools/actions?query=workflow%3ACI)
+[![codecov](https://codecov.io/gh/jazzband/pip-tools/branch/main/graph/badge.svg)](https://codecov.io/gh/jazzband/pip-tools)
+
+# pip-tools = pip-compile + pip-sync
+
+A set of command line tools to help you keep your `pip`-based packages fresh,
+even when you've pinned them. You do pin them, right? (In building your Python application and its dependencies for production, you want to make sure that your builds are predictable and deterministic.)
+
+[![pip-tools overview for phase II](https://github.com/jazzband/pip-tools/raw/main/img/pip-tools-overview.svg)](https://github.com/jazzband/pip-tools/raw/main/img/pip-tools-overview.svg)
+
+## Installation
+
+Similar to `pip`, `pip-tools` must be installed in each of your project's
+[virtual environments](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments):
+
+```
```

Comment:
For code snippets with shell prompts and command output, always use the `console` highligher:
````suggestion
```console
````